<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dubheClass</class>
 <widget class="QMainWindow" name="dubheClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dubhe</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="indexBackWdt" native="true">
      <layout class="QGridLayout" name="gridLayout_9">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QWidget" name="indexTopWdt" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="logoLr">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="yoloLr">
                  <property name="text">
                   <string>yolo标注工具</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <property name="spacing">
                 <number>15</number>
                </property>
                <property name="rightMargin">
                 <number>20</number>
                </property>
                <item>
                 <widget class="QPushButton" name="minBtn">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="maxBtn">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="closeBtn">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="8,3">
              <item>
               <widget class="QWidget" name="showWdt" native="true">
                <layout class="QGridLayout" name="gridLayout_6">
                 <item row="0" column="0">
                  <widget class="QScrollArea" name="showSa">
                   <property name="widgetResizable">
                    <bool>true</bool>
                   </property>
                   <widget class="QWidget" name="scrollAreaWidgetContents">
                    <property name="geometry">
                     <rect>
                      <x>0</x>
                      <y>0</y>
                      <width>616</width>
                      <height>697</height>
                     </rect>
                    </property>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QWidget" name="widget_4" native="true">
                  <layout class="QGridLayout" name="gridLayout_3">
                   <item row="0" column="0">
                    <layout class="QVBoxLayout" name="verticalLayout" stretch="4,6">
                     <item>
                      <widget class="QWidget" name="widget_5" native="true">
                       <layout class="QGridLayout" name="gridLayout_5">
                        <property name="leftMargin">
                         <number>0</number>
                        </property>
                        <property name="rightMargin">
                         <number>0</number>
                        </property>
                        <item row="0" column="0">
                         <layout class="QVBoxLayout" name="verticalLayout_3">
                          <item>
                           <layout class="QHBoxLayout" name="horizontalLayout_3">
                            <item>
                             <widget class="QLabel" name="flagLr">
                              <property name="minimumSize">
                               <size>
                                <width>4</width>
                                <height>16</height>
                               </size>
                              </property>
                              <property name="maximumSize">
                               <size>
                                <width>4</width>
                                <height>16</height>
                               </size>
                              </property>
                              <property name="text">
                               <string/>
                              </property>
                             </widget>
                            </item>
                            <item>
                             <widget class="QLabel" name="curFlagLr">
                              <property name="text">
                               <string>当前标签</string>
                              </property>
                             </widget>
                            </item>
                            <item>
                             <spacer name="horizontalSpacer_2">
                              <property name="orientation">
                               <enum>Qt::Horizontal</enum>
                              </property>
                              <property name="sizeHint" stdset="0">
                               <size>
                                <width>40</width>
                                <height>20</height>
                               </size>
                              </property>
                             </spacer>
                            </item>
                           </layout>
                          </item>
                          <item>
                           <widget class="QTableView" name="labelsLv">
                            <property name="verticalScrollBarPolicy">
                             <enum>Qt::ScrollBarAlwaysOff</enum>
                            </property>
                            <property name="horizontalScrollBarPolicy">
                             <enum>Qt::ScrollBarAlwaysOff</enum>
                            </property>
                            <property name="showGrid">
                             <bool>false</bool>
                            </property>
                            <attribute name="horizontalHeaderCascadingSectionResizes">
                             <bool>true</bool>
                            </attribute>
                            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                             <bool>false</bool>
                            </attribute>
                            <attribute name="horizontalHeaderStretchLastSection">
                             <bool>false</bool>
                            </attribute>
                            <attribute name="verticalHeaderVisible">
                             <bool>false</bool>
                            </attribute>
                           </widget>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </widget>
                     </item>
                     <item>
                      <widget class="QWidget" name="widget_6" native="true">
                       <layout class="QGridLayout" name="gridLayout_4">
                        <property name="leftMargin">
                         <number>0</number>
                        </property>
                        <property name="rightMargin">
                         <number>0</number>
                        </property>
                        <item row="0" column="0">
                         <layout class="QVBoxLayout" name="verticalLayout_2">
                          <item>
                           <layout class="QHBoxLayout" name="horizontalLayout_2">
                            <property name="leftMargin">
                             <number>6</number>
                            </property>
                            <item>
                             <widget class="QLabel" name="picPath2Lr">
                              <property name="minimumSize">
                               <size>
                                <width>4</width>
                                <height>16</height>
                               </size>
                              </property>
                              <property name="maximumSize">
                               <size>
                                <width>4</width>
                                <height>16</height>
                               </size>
                              </property>
                              <property name="text">
                               <string/>
                              </property>
                             </widget>
                            </item>
                            <item>
                             <widget class="QLabel" name="picPath1Lr">
                              <property name="text">
                               <string>图片路径</string>
                              </property>
                             </widget>
                            </item>
                            <item>
                             <widget class="QLabel" name="picPathLr">
                              <property name="text">
                               <string>d:\test\pic</string>
                              </property>
                             </widget>
                            </item>
                            <item>
                             <spacer name="horizontalSpacer">
                              <property name="orientation">
                               <enum>Qt::Horizontal</enum>
                              </property>
                              <property name="sizeHint" stdset="0">
                               <size>
                                <width>40</width>
                                <height>20</height>
                               </size>
                              </property>
                             </spacer>
                            </item>
                            <item>
                             <widget class="QPushButton" name="openDirBtn">
                              <property name="minimumSize">
                               <size>
                                <width>110</width>
                                <height>36</height>
                               </size>
                              </property>
                              <property name="maximumSize">
                               <size>
                                <width>110</width>
                                <height>36</height>
                               </size>
                              </property>
                              <property name="text">
                               <string>打开图片目录</string>
                              </property>
                             </widget>
                            </item>
                           </layout>
                          </item>
                          <item>
                           <widget class="QListWidget" name="picListLwt">
                            <property name="verticalScrollBarPolicy">
                             <enum>Qt::ScrollBarAlwaysOff</enum>
                            </property>
                            <property name="horizontalScrollBarPolicy">
                             <enum>Qt::ScrollBarAlwaysOff</enum>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_2" native="true">
                  <layout class="QGridLayout" name="gridLayout_2">
                   <item row="0" column="0">
                    <layout class="QHBoxLayout" name="horizontalLayout">
                     <property name="spacing">
                      <number>20</number>
                     </property>
                     <item>
                      <widget class="QPushButton" name="datasetBtn">
                       <property name="minimumSize">
                        <size>
                         <width>102</width>
                         <height>36</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>102</width>
                         <height>36</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>划分数据集</string>
                       </property>
                       <property name="icon">
                        <iconset resource="dubhe.qrc">
                         <normaloff>:/dubhe/pic/dataset.png</normaloff>:/dubhe/pic/dataset.png</iconset>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="trainBtn">
                       <property name="minimumSize">
                        <size>
                         <width>102</width>
                         <height>36</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>102</width>
                         <height>36</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>训练模型</string>
                       </property>
                       <property name="icon">
                        <iconset resource="dubhe.qrc">
                         <normaloff>:/dubhe/pic/train.png</normaloff>:/dubhe/pic/train.png</iconset>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="helpBtn">
                       <property name="minimumSize">
                        <size>
                         <width>66</width>
                         <height>36</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>66</width>
                         <height>36</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>帮助</string>
                       </property>
                       <property name="icon">
                        <iconset resource="dubhe.qrc">
                         <normaloff>:/dubhe/pic/help.png</normaloff>:/dubhe/pic/help.png</iconset>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="exitBtn">
                       <property name="minimumSize">
                        <size>
                         <width>66</width>
                         <height>36</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>66</width>
                         <height>36</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>退出</string>
                       </property>
                       <property name="icon">
                        <iconset resource="dubhe.qrc">
                         <normaloff>:/dubhe/pic/exit.png</normaloff>:/dubhe/pic/exit.png</iconset>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dubhe.qrc"/>
 </resources>
 <connections/>
</ui>
